paasta_tools/cli/fsm/questions.py

```diff
@@ -121,37 +120,23 @@ def get_deploy_stanza():
         {"instancename": "security-check", },
         {"instancename": "push-to-registry", },
         {"instancename": "performance-check", },
-        {"instancename": "pnw-stagea.canary", },
-        {"instancename": "pnw-stagea.main", },
-        {"instancename": "norcal-stageb.canary", },
-        {"instancename": "norcal-stageb.main", },
-        {"instancename": "norcal-devb.canary", },
-        {"instancename": "norcal-devb.main", },
-        {"instancename": "norcal-devc.canary", },
-        {"instancename": "norcal-devc.main", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.canary", },
-        {"instancename": "nova-prod.canary", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.main", },
-        {"instancename": "nova-prod.main", },
+        {"instancename": "dev.everything", "trigger_next_step_manually": True, },
+        {"instancename": "stage.everything", "trigger_next_step_manually": True, },
+        {"instancename": "prod.canary", "trigger_next_step_manually": True, },
+        {"instancename": "prod.non_canary", },
     ]
     return stanza
 
 
-def get_clusternames_from_deploy_stanza(deploy_stanza):
-    """Given a dictionary deploy_stanza, as from get_deploy_stanza(), return a
-    set of the clusternames referenced in the pipeline.
-    """
-    clusternames = set()
-    for entry in deploy_stanza.get("pipeline", []):
-        instancename = entry["instancename"]
-        if instancename in DEPLOY_PIPELINE_NON_DEPLOY_STEPS:
-            continue
-        # Usually clustername will be instancename.namespace, so lop off
-        # namespace. (If there's no namespace, this just returns clustername,
-        # which is correct.)
-        clustername = instancename.split(".")[0]
-        clusternames.add(clustername)
-    return clusternames
+def get_default_clusternames():
```

Comment:
Can you change this to get_default_yelp_clusternames? To be more explicit this is a yelp-specific setting. (yuck)

Or if you feel adventurous, I can show you how to deploy a config file (easy) and have it read from that instead and we'll have one less yelpism.